OWML.ModHelper.Menus/ModMenuWithSelectables.cs

```diff
@@ -151,13 +149,13 @@ protected virtual void AddSelectable(Selectable selectable)
 
         protected virtual void OnActivateMenu()
         {
-            CommandListener.gameObject.SetActive(true);
-            CommandListener.BlockNextRelease();
+            //CommandListener.gameObject.SetActive(true); // todo hmmmm
+            _events.Input.BlockNextRelease();
         }
 
         protected virtual void OnDeactivateMenu()
         {
-            CommandListener.gameObject.SetActive(false);
+            //CommandListener.gameObject.SetActive(false); // todo hmmmm
```

Comment:
and here

Reply:
ooh. BlockNextRelease probably have to be scrapped now